ckanext/cfpb_extrafields/controllers/tst_import.py

```diff
@@ -0,0 +1,6 @@
+from ckanext.cfpb_extrafields.digutils import make_rec
+
+if __name__ == "__main__":
+    with open("/home/cfpb/boehmm/dig.xlsm", "r") as f:
```

Comment:
Yes, I should probably remove this entire file for the time being. There's unit tests for the pure function, but if there's a dig file that folks feel comfortable putting on the public github, I could make unit tests around that.